Docker/itk-base/Dockerfile

```diff
@@ -16,7 +17,11 @@ RUN curl -O https://cmake.org/files/v3.4/cmake-3.4.3.tar.gz && \
     cd /tmp/cmake-build && \
     ../cmake-3.4.3/bootstrap && \
     make -j$(nproc) && \
-    ./bin/cmake -DCMAKE_BUILD_TYPE:STRING=Release . && \
+    cd .. && \
```

Comment:
`pip install cmake` fails on the current base image (`debian jessie`):
`root@a31dc352c1c8:/# pip  install skbuild
Downloading/unpacking skbuild
  Could not find any downloads that satisfy the requirement skbuild
Cleaning up...
No distributions at all found for skbuild
Storing debug log for failure in /root/.pip/pip.log`